DatabaseManager.py

```diff
@@ -0,0 +1,87 @@
+import sqlite3
+import logging
+import pandas as pd
+from typing import Dict
+from pathlib import Path
+
+class DatabaseManager:
+    def __init__(self, db_path: str = "spotify_db/spotify.db", schema_dir: str = "spotify_db/schema"):
+        self.db_path = db_path
+        self.schema_dir = schema_dir
+        self.connection = None
+        self._initialize_database()
+
+    def _load_schema_file(self, filename: str) -> str:
+        schema_path = Path(self.schema_dir) / filename
+        with open(schema_path, 'r', encoding='utf-8') as f:
+            return f.read()
+
+    def _initialize_database(self):
+        self.connection = sqlite3.connect(self.db_path)
+        cursor = self.connection.cursor()
+        cursor.execute("PRAGMA foreign_keys = OFF")
+
+        schema_files = [
+            'create_artist.sql',
+            'create_user.sql',
+            'create_album.sql',
+            'create_track.sql',
+            'create_playlist.sql',
+            'create_playlist_track.sql'
+        ]
```

Comment:
Instead of listing out the schema files manually, can you find a way to automatically get a list of all `.sql` files in the `schema_dir`?